readme/LandingPage.scalatex

```diff
@@ -0,0 +1,328 @@
+      @b
+        Dialects.
+      Scalameta is designed from the ground up to understand different versions of the base language:
+      Scala 2.10, Scala 2.11, Scala 2.12 and even Dotty. We also support Sbt build files
+      to make sure we cover as much Scala code as possible.
+
+    @meta
+      import scala.meta.dialects.Sbt0137
+      Sbt0137("""
+        lazy val root = (project in file(".")).
+        settings(name := "hello")
+      """).parse[Source]
+    @table
+      @(cls := "pure-table")
+      @(margin := "auto")
+      @thead
+        @tr
+          @th
+            Dialect
+          @th
+            Syntax
+          @th
+            Semantic
+          @th
+            Notes
+      @tbody
+        @tr
+          @td
+            Scala 2.10
+          @td
+            Yes
+          @td
+            Yes*
+          @td
+            Requires @lnk("an external module", "https://github.com/scalameta/sbthost")
+        @tr
+          @td
+            Scala 2.11
+          @td
+            Yes
+          @td
+            Full support
+          @td{}
+        @tr
+          @td
+            Scala 2.12
+          @td
+            Yes
+          @td
+            Full support
+          @td{}
+        @tr
+          @td
+            Dotty
+          @td
+            Partial
+          @td
+            No
+          @td
+            Partial support for new language features.
+        @tr
+          @td
+            Sbt 0.13
+          @td
+            Yes
+          @td
+            Yes*
+          @td
+            Requires @lnk("an external module", "https://github.com/scalameta/sbthost")
+        @tr
+          @td
+            Sbt 1.0
+          @td
+            Yes
+          @td
+            No
+          @td{}
+
+    @p
+      @b
+        Semantic API.
+      Semantic analysis of Scala programs has typically been tricky business, requiring
+      intimate familiarity with compiler internals.
+      Scalameta involves none of that. Our semantic API is powered by
+      @lnk("Semantic DB", "http://scalameta.org/tutorial/#SemanticDB"), a simple
+      @lnk("schema", "https://github.com/scalameta/scalameta/blob/master/langmeta/semanticdb/shared/src/main/protobuf/semanticdb.proto")
+      for persisting semantic information extracted from the Scala compiler.
+      Saved semantic information can then be used in multiple applications running on
+      different platforms, potentially in a distributed fashion.
+      @table
+        @(cls := "pure-table")
+        @(margin := "auto")
+        @thead
+          @tr
+            @th
+              Feature
+            @th
+              Supported?
+        @tbody
+          @def feature(name: String, supported: String) = tr(td(name), td(supported))
+          @feature("Compiler messages", "Yes")
+          @feature("Symbol at position", "Yes")
+          @feature("Symbol signature", "Yes")
+          @feature("Type at position", "No")
+          @feature("Type members (completion)", "No")
```

Comment:
Is this really equivalent to completion though?

Reply:
I added "Auto completion -> No" in a separate row.